Source/Core/DolphinWX/InputConfigDiag.h

```diff
@@ -83,6 +84,38 @@ class PadSettingCheckBox : public PadSetting
 	ControllerEmu::ControlGroup::Setting* const setting;
 };
 
+class InputEventFilter : public wxEventFilter
+{
+public:
+	InputEventFilter()
+		: m_block(false)
+	{
+		wxEvtHandler::AddFilter(this);
+	};
+
+	~InputEventFilter()
+	{
+		wxEvtHandler::RemoveFilter(this);
+	};
+
+	int FilterEvent(wxEvent& event) override;
+
+	void BlockEvents(bool block) { m_block = block; };
+
+private:
+	static inline bool ShouldCatchEventType(wxEventType type)
```

Comment:
Have you checked if inline actually does anything? (hint: It doesn't, functions in a class definition are implicitly inline by default).